<ui version="4.0" >
 <class>roomedit_dialog</class>
 <widget class="QDialog" name="roomedit_dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>577</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>838</width>
    <height>559</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16000</width>
    <height>16000</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Room Information Editor</string>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>560</x>
     <y>540</y>
     <width>351</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="spacing" >
     <number>6</number>
    </property>
    <property name="leftMargin" >
     <number>0</number>
    </property>
    <property name="topMargin" >
     <number>0</number>
    </property>
    <property name="rightMargin" >
     <number>0</number>
    </property>
    <property name="bottomMargin" >
     <number>0</number>
    </property>
    <item>
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" >
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okButton" >
      <property name="text" >
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton" >
      <property name="text" >
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" native="1" name="widget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>60</y>
     <width>899</width>
     <height>229</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" native="1" name="widget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" native="1" name="widget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>899</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" native="1" name="widget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>30</y>
     <width>897</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>60</y>
     <width>901</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="spacing" >
     <number>6</number>
    </property>
    <property name="leftMargin" >
     <number>0</number>
    </property>
    <property name="topMargin" >
     <number>0</number>
    </property>
    <property name="rightMargin" >
     <number>0</number>
    </property>
    <property name="bottomMargin" >
     <number>0</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox_2" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
        <horstretch>41</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Description</string>
      </property>
      <widget class="QTextEdit" name="textEdit_desc" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>20</y>
         <width>701</width>
         <height>201</height>
        </rect>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>10</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
        <horstretch>10</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Preferences</string>
      </property>
      <widget class="QWidget" name="layoutWidget_4_2" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>20</y>
         <width>156</width>
         <height>187</height>
        </rect>
       </property>
       <layout class="QGridLayout" >
        <property name="leftMargin" >
         <number>0</number>
        </property>
        <property name="topMargin" >
         <number>0</number>
        </property>
        <property name="rightMargin" >
         <number>0</number>
        </property>
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <property name="horizontalSpacing" >
         <number>6</number>
        </property>
        <property name="verticalSpacing" >
         <number>6</number>
        </property>
        <item row="1" column="0" colspan="2" >
         <widget class="QComboBox" name="comboBox_terrain" />
        </item>
        <item row="5" column="0" >
         <widget class="QLabel" name="label_16_2" >
          <property name="text" >
           <string>Z</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QLineEdit" name="lineEdit_coordx" />
        </item>
        <item row="4" column="0" >
         <widget class="QLabel" name="label_17_2" >
          <property name="text" >
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="label_2_2" >
          <property name="text" >
           <string>Terrain</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" >
         <widget class="QLineEdit" name="lineEdit_coordy" />
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="label_15_2" >
          <property name="text" >
           <string>X</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1" >
         <widget class="QLineEdit" name="lineEdit_coordz" />
        </item>
        <item row="2" column="0" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>0</y>
     <width>901</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" >
    <property name="spacing" >
     <number>6</number>
    </property>
    <property name="leftMargin" >
     <number>0</number>
    </property>
    <property name="topMargin" >
     <number>0</number>
    </property>
    <property name="rightMargin" >
     <number>0</number>
    </property>
    <property name="bottomMargin" >
     <number>0</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox_3" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
        <horstretch>20</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Room Name</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_name" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>20</y>
         <width>751</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_5" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Room ID</string>
      </property>
      <widget class="QLabel" name="label_roomid" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font" >
        <font>
         <family>Sans Serif</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text" >
        <string>ID</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignHCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>290</y>
     <width>361</width>
     <height>241</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
     <horstretch>12</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title" >
    <string>Note</string>
   </property>
   <widget class="QTextEdit" name="textEdit_note" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
      <horstretch>1</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3_2" >
   <property name="geometry" >
    <rect>
     <x>380</x>
     <y>290</y>
     <width>531</width>
     <height>241</height>
    </rect>
   </property>
   <property name="sizePolicy" >
    <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
     <horstretch>13</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title" >
    <string>Exits</string>
   </property>
   <widget class="QWidget" native="1" name="layoutWidget_4_2_2_2_2_2" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>167</width>
      <height>182</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_S" >
    <property name="geometry" >
     <rect>
      <x>11</x>
      <y>117</y>
      <width>33</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>S</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_W" >
    <property name="geometry" >
     <rect>
      <x>11</x>
      <y>148</y>
      <width>33</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>W</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_S" >
    <property name="geometry" >
     <rect>
      <x>439</x>
      <y>115</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_U" >
    <property name="geometry" >
     <rect>
      <x>439</x>
      <y>177</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_E" >
    <property name="geometry" >
     <rect>
      <x>439</x>
      <y>84</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_D" >
    <property name="geometry" >
     <rect>
      <x>439</x>
      <y>208</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>339</x>
      <y>21</y>
      <width>94</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text" >
     <string>Leads to</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_N" >
    <property name="geometry" >
     <rect>
      <x>11</x>
      <y>55</y>
      <width>33</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>N</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_U" >
    <property name="geometry" >
     <rect>
      <x>11</x>
      <y>179</y>
      <width>33</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>U</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_E" >
    <property name="geometry" >
     <rect>
      <x>11</x>
      <y>86</y>
      <width>33</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>E</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_N" >
    <property name="geometry" >
     <rect>
      <x>439</x>
      <y>53</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_D" >
    <property name="geometry" >
     <rect>
      <x>11</x>
      <y>210</y>
      <width>33</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>D</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_W" >
    <property name="geometry" >
     <rect>
      <x>439</x>
      <y>146</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>439</x>
      <y>21</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text" >
     <string>Flag</string>
    </property>
   </widget>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>50</x>
      <y>21</y>
      <width>283</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text" >
     <string>Door</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_doordown" >
    <property name="geometry" >
     <rect>
      <x>50</x>
      <y>208</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
      <horstretch>2</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_doorup" >
    <property name="geometry" >
     <rect>
      <x>50</x>
      <y>177</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
      <horstretch>2</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_doorwest" >
    <property name="geometry" >
     <rect>
      <x>50</x>
      <y>146</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
      <horstretch>2</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_doorsouth" >
    <property name="geometry" >
     <rect>
      <x>50</x>
      <y>115</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
      <horstretch>2</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_dooreast" >
    <property name="geometry" >
     <rect>
      <x>50</x>
      <y>84</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
      <horstretch>2</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_doornorth" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>50</x>
      <y>53</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
      <horstretch>3</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_leadN" >
    <property name="geometry" >
     <rect>
      <x>292</x>
      <y>53</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="sizePolicy" >
     <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
      <horstretch>1</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_leadE" >
    <property name="geometry" >
     <rect>
      <x>292</x>
      <y>84</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_leadS" >
    <property name="geometry" >
     <rect>
      <x>292</x>
      <y>115</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_leadW" >
    <property name="geometry" >
     <rect>
      <x>292</x>
      <y>146</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_leadU" >
    <property name="geometry" >
     <rect>
      <x>292</x>
      <y>177</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_leadD" >
    <property name="geometry" >
     <rect>
      <x>292</x>
      <y>208</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>roomedit_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>roomedit_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
